pkg/alertmanager/amcfg_test.go

```diff
@@ -41,8 +42,17 @@ import (
 	"github.com/prometheus-operator/prometheus-operator/pkg/assets"
 )
 
+func mustMarshalRoute(r monitoringv1alpha1.Route) []byte {
+	b, err := json.Marshal(r)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
What do you think about adding `t *testing.T` as an argument here and using `t.Fatal` or `require.NoError` instead of panic?